gui.js

```diff
@@ -4472,22 +4511,6 @@ IDE_Morph.prototype.getURL = function (url) {
     }
 };
 
-IDE_Morph.prototype.getURLsbeOrRelative = function (url) {
```

Comment:
Right now, all the loading is still XHR on a `file://` so it won't work. 
It's been broken since #814 

However, everyone who's downloaded Snap<i>!</i> locally will have all the necessary files, so even if they were offline the menus wouldn't work. I don't think there's much use here to have the fallback. (All the fallback was supposed to do was just attempt to load from s.b.e/X if something failed.)